docs/robot_protocols.md

```diff
@@ -2,16 +2,14 @@
 title: Tn5 RNA-Seq Liquid-handling Robot Protocols
 ---
 
-## Liquid-handling Robot Protocols
+\| [Home](index) \| [Analysis Pipeline](pipeline) \| [mRNA Extraction Protocol](mrna_extraction) \| [TM3'seq protocol](tm3seq_protocol) \| [Liquid-Handling Robot Protocols](robot_protocols) \|
 
-\| [Home](index) \| [Analysis Pipeline](pipeline) \| [mRNA Extraction](mrna_extraction) \| [TM3'seq protocol](tm3seq_protocol) \| [Liquid-Handling Robot Protocols](robot_protocols) \|
+## Liquid-handling robot protocols
 
-1.  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur rutrum enim sit amet scelerisque aliquet. Vivamus leo nulla, imperdiet accumsan tempus in, feugiat eget erat. Mauris congue nec nulla id ultrices. Suspendisse consequat ligula vitae quam lobortis, ultricies aliquet mauris vestibulum. Donec rutrum faucibus dui, a efficitur nisi aliquet in. Praesent a bibendum quam. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Cras eros nulla, tempor eu convallis fermentum, sollicitudin in risus. Quisque in dui nec leo posuere dapibus sit amet eu nisi. Nam ut risus vel mi maximus fringilla non eu turpis. Nam tempor dignissim nulla vitae mollis. Etiam sed risus erat. Sed egestas nisi nunc, ac vehicula mi facilisis eu. Curabitur vel lacus enim. Donec nibh nulla, tempor sit amet imperdiet ut, sollicitudin ornare orci. Phasellus ac massa at dolor posuere consectetur. 
+The mRNA extraction and TM3'seq protocols were optimized to work on 96-well plates, and therefore they can be used as reference to develop protocols for liquid-handling platforms. 
 
-2.  Duis magna urna, elementum at justo nec, laoreet porta purus. Sed ut metus congue, imperdiet ipsum a, lobortis turpis. Cras aliquet orci quis est condimentum, sed lobortis leo bibendum. Pellentesque erat ante, pretium eu tempus nec, pretium non elit. Duis finibus nisl a quam cursus placerat. Nam tincidunt diam turpis, nec placerat orci ultricies id. Morbi sed dictum ligula. Nunc porttitor iaculis elit ut vehicula. Morbi vehicula est et facilisis mollis. Praesent vel volutpat ex, eget mattis mauris. Duis faucibus elementum velit, a ullamcorper mauris consequat id. Aliquam posuere ipsum eu luctus auctor.
+We have implemented both protocols in CyBio® FeliX pipetting robot (Analitik Jena). The main protocols, subroutines, and set-up files for this specific liquid-handling platform are available under request. Please contact the authors: 
```

Comment:
I presume this will be made available on the website somewhere before publication?

Reply:
I don't really want to go through that. I tried to put the protocols together, but besides the main protocol there are too many subroutines, etc, so it becomes too many files too quickly. I think is OK if people that have our same specific robot request the protocols. I've shared them already with some people, and that way I feel comfortable with sharing them the way they are. For the page it will require a looot of work. Doable, but maybe not necessary. Let's see.

Reply:
That makes sense, but perhaps offer whatever you have in a downloadable zip file?